v2/src/components/tabs/OAuthMobileTabs.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+let Tabs = require("@theme/Tabs").default;
+
+export default function OAuthMobileTabs(props: any) {
```

Comment:
We already have a component to render the mobile code tabs. Please reuse that.

Reply:
I don't think I can use that. It's a subtabs component. I've updated the oauth tabs to match the options that we expose in those subtabs